AddOns/BasicFinancials/app/Translations/Basic Financials.g.xlf

```diff
@@ -0,0 +1,174 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
+  <file datatype="xml" source-language="en-US" target-language="en-US" original="Basic Financials">
+    <body>
+      <group id="body">
+        <trans-unit id="Codeunit 875245055 - NamedType 1420859235" size-unit="char" translate="yes" xml:space="preserve">
+          <source>Set up Basic Financials extension</source>
+          <note from="Developer" annotates="general" priority="2"></note>
+          <note from="Xliff Generator" annotates="general" priority="3">Codeunit Assisted Setup BF - NamedType AssistedSetupTxt</note>
+        </trans-unit>
+        <trans-unit id="Codeunit 875245055 - NamedType 2493814900" size-unit="char" translate="no" xml:space="preserve">
+          <source>https://go.microsoft.com/fwlink/?linkid=</source>
+          <note from="Developer" annotates="general" priority="2"></note>
+          <note from="Xliff Generator" annotates="general" priority="3">Codeunit Assisted Setup BF - NamedType AssistedSetupHelpTxt</note>
+        </trans-unit>
+        <trans-unit id="Codeunit 875245055 - NamedType 1689235809" size-unit="char" translate="yes" xml:space="preserve">
+          <source>Basic Financials</source>
+          <note from="Developer" annotates="general" priority="2"></note>
+          <note from="Xliff Generator" annotates="general" priority="3">Codeunit Assisted Setup BF - NamedType AssistedSetupDescriptionTxt</note>
+        </trans-unit>
+        <trans-unit id="Codeunit 875245055 - NamedType 2007470713" size-unit="char" translate="yes" xml:space="preserve">
+          <source>You have already completed the Basic Financials extension assisted setup guide. Do you want to run it again ?</source>
```

Comment:
This assisted setup guide has already been completed. Do you want to run it again?